packages/core/parcel-bundler/src/assets/CSSAsset.js

```diff
@@ -125,6 +126,30 @@ class CSSAsset extends Asset {
         'module.exports = ' + JSON.stringify(this.cssModules, null, 2) + ';';
     }
 
+    let map;
+    if (this.options.sourceMaps && this.sourceMap) {
+      const source = this.contents.split('\n');
+      const mappings = this.sourceMap._mappings._array
+        .map(v => ({
+          source: this.relativeName,
+          original: {
+            line: v.originalLine,
+            column: v.originalColumn
+          },
+          generated: {
+            line: v.generatedLine,
+            column: v.generatedColumn
+          }
+        }))
+        .filter(
+          ({original: {line, column}}) =>
+            line - 1 < source.length && column < source[line - 1].length
```

Comment:
PostCSS seems to generate sourcemaps with invalid source positions: [with this visualizer](http://sokra.github.io/source-map-visualization/#base64,ZGl2e3dpZHRoOjEwMHB4O2hlaWdodDoxMDBweH1tYWlue2ZvbnQtZmFtaWx5Om1vbm9zcGFjZX1ib2R5e2JhY2tncm91bmQtY29sb3I6cmVkfQovKiMgc291cmNlTWFwcGluZ1VSTD0vc3R5bGUuY3NzLm1hcCov,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIm90aGVyLXN0eWxlLmNzcyIsImFub3RoZXItc3R5bGUuY3NzIiwic3R5bGUuY3NzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQUNBLElBQ0MsWUFBYSxBQUNiLFlBQWMsQ0FDZCxBQ0pELEtBQ0MscUJBQXVCLENBQ3ZCLEFDQ0QsS0FDQyxvQkFBc0IsQ0FDdEIiLCJmaWxlIjoic3R5bGUuY3NzLm1hcCIsInNvdXJjZVJvb3QiOiIuLi9pbnRlZ3JhdGlvbi9zb3VyY2VtYXAtY3NzLWltcG9ydCIsInNvdXJjZXNDb250ZW50IjpbIlxuZGl2IHtcblx0d2lkdGg6IDEwMHB4O1xuXHRoZWlnaHQ6IDEwMHB4O1xufVxuIiwibWFpbiB7XG5cdGZvbnQtZmFtaWx5OiBtb25vc3BhY2U7XG59XG4iLCJcbkBpbXBvcnQgXCIuL290aGVyLXN0eWxlLmNzc1wiO1xuXG5ib2R5IHtcblx0YmFja2dyb3VuZC1jb2xvcjogcmVkO1xufVxuXG5AaW1wb3J0IFwiLi9hbm90aGVyLXN0eWxlLmNzc1wiO1xuIl19,CmRpdiB7Cgl3aWR0aDogMTAwcHg7CgloZWlnaHQ6IDEwMHB4Owp9Cg==,bWFpbiB7Cglmb250LWZhbWlseTogbW9ub3NwYWNlOwp9Cg==,CkBpbXBvcnQgIi4vb3RoZXItc3R5bGUuY3NzIjsKCmJvZHkgewoJYmFja2dyb3VuZC1jb2xvcjogcmVkOwp9CgpAaW1wb3J0ICIuL2Fub3RoZXItc3R5bGUuY3NzIjsK), these mappings are clearly visible (by hovering over the highlighted entries in the bottom section).
With the filtering, [this](http://sokra.github.io/source-map-visualization/#base64,ZGl2e3dpZHRoOjEwMHB4O2hlaWdodDoxMDBweH1tYWlue2ZvbnQtZmFtaWx5Om1vbm9zcGFjZX1ib2R5e2JhY2tncm91bmQtY29sb3I6cmVkfQovKiMgc291cmNlTWFwcGluZ1VSTD0vc3R5bGUuY3NzLm1hcCov,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIm90aGVyLXN0eWxlLmNzcyIsImFub3RoZXItc3R5bGUuY3NzIiwic3R5bGUuY3NzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQUNBLElBQ0MsWUFDQSxhQ0hELEtBQ0Msc0JDRUQsS0FDQyIsImZpbGUiOiJzdHlsZS5jc3MubWFwIiwic291cmNlUm9vdCI6Ii4uL2ludGVncmF0aW9uL3NvdXJjZW1hcC1jc3MtaW1wb3J0Iiwic291cmNlc0NvbnRlbnQiOlsiXG5kaXYge1xuXHR3aWR0aDogMTAwcHg7XG5cdGhlaWdodDogMTAwcHg7XG59XG4iLCJtYWluIHtcblx0Zm9udC1mYW1pbHk6IG1vbm9zcGFjZTtcbn1cbiIsIlxuQGltcG9ydCBcIi4vb3RoZXItc3R5bGUuY3NzXCI7XG5cbmJvZHkge1xuXHRiYWNrZ3JvdW5kLWNvbG9yOiByZWQ7XG59XG5cbkBpbXBvcnQgXCIuL2Fub3RoZXItc3R5bGUuY3NzXCI7XG4iXX0=,CmRpdiB7Cgl3aWR0aDogMTAwcHg7CgloZWlnaHQ6IDEwMHB4Owp9Cg==,bWFpbiB7Cglmb250LWZhbWlseTogbW9ub3NwYWNlOwp9Cg==,CkBpbXBvcnQgIi4vb3RoZXItc3R5bGUuY3NzIjsKCmJvZHkgewoJYmFja2dyb3VuZC1jb2xvcjogcmVkOwp9CgpAaW1wb3J0ICIuL2Fub3RoZXItc3R5bGUuY3NzIjsK) is the output.

Chrome works with both cases, but the `source-map` package used in the tests doesn't behave as expected with the duplicate & incorrect mapppings.